### DISCUSSION TITLE
Submit button language settings

### DISCUSSION THREAD

**Discussion by teslum369** [cpeditor/cpeditor#821] [Q&A] [answered]
Submit button in CP editor automatically sets the lang as gnu c++11, how can we change the settings of it to e.g. c++17?

**Comment by coder3101** [comment, marked as answer]
While setting up CF template you must have chosen the language for submission. You can edit or create a new template and choose c++17 compiler. Make sure to delete the old config. You should not have two template for same language.

**Reply by teslum369** [reply]
Thank you very much. One more question. Is there possibility to set this lang?: GNU G++17 9.0.2 (64 bit, msys 2)?
I can't really see it on the list when configuring a template :/

**Reply by coder3101** [reply]
If it helps you, mark it as answer.

**Reply by ouuan** [reply]
> Is there possibility to set this lang?: GNU G++17 9.0.2 (64 bit, msys 2)?
> I can't really see it on the list when configuring a template :/

It's probably a CF Tool issue. Unfortunately, CF Tool is unmaintained, but in the future CP Editor will switch to other tools or let the user choose the tool to use.

**Reply by teslum369** [reply]
Great!

Thank you for the help and also I wish you and the team all the best. Good luck :)
